.moai/reports/v311-docs/pr-followup.md

```
@@ -0,0 +1,37 @@
+## Follow-up audit — four stale claims corrected, one of them mine
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)